libs/expo/betterangels/src/lib/screens/Note/NoteServices.tsx

```diff
@@ -100,17 +49,14 @@ const ICONS = [
 const getIcons = (
   array: {
     id: string;
-    descriptor?: MoodEnum;
     service?: ServiceEnum;
     customService?: string | null | undefined;
   }[]
 ) => {
   return array.slice(0, 4).map((icon) => {
     let iconEnumValue = '';
 
-    if ('descriptor' in icon) {
-      iconEnumValue = icon.descriptor as string;
-    } else if ('service' in icon) {
+    if ('service' in icon) {
```

Comment:
if want, can get rid of the `if` statement with `let iconEnumValue = icon.service || '';`